src/util/redisConnector.js

```diff
@@ -0,0 +1,36 @@
+const redis = require('redis');
+const log = require('../logger');
+
+let a = 1;
+const host = process.env.REDIS_HOST || 'localhost';
+const port = process.env.REDIS_PORT || 6379;
+const password = process.env.REDIS_PASSWORD || '';
+const DBConnector = {
+  redisInstance: redis.createClient({
```

Comment:
What if `createClient` throws error how will we handle that?

Reply:
It will be catched in try catch block where we are invoking this class. Example: https://github.com/rudderlabs/rudder-transformer/blob/56b90cf1f66efdc9aeddf4ff68274c2f7692c6b5/src/v0/sources/shopify/transform.js#L216